uber/templates/signups/shifts.html

```diff
@@ -87,22 +87,17 @@
                 shiftList.push({
                     title : "{{ job.name }} ({{ job.location_label }})",
                     start : "{{ job.start_time_local|datetime("%Y-%m-%dT%H:%M:%S") }}",
-                    end : "{{job.end_time_local|datetime("%Y-%m-%dT%H:%M:%S") }}",
-                    id : "{{job.id}}",
-                    {% if job.taken %}
-                        backgroundColor: "#239875",
-                        taken: true
-                    {% else %}
-                        backgroundColor: "#305fc9",
-                        taken: false
-                    {% endif %}
+                    end : "{{ job.end_time_local|datetime("%Y-%m-%dT%H:%M:%S") }}",
```

Comment:
This is just some clean-up. The shiftList is the list of shifts that are taken by the attendee... so there's no reason to check for `job.taken` a second time.